hummingbot/strategy_v2/executors/position_executor/position_executor.py

```diff
@@ -280,7 +280,7 @@ async def control_shutdown_process(self):
 
         :return: None
         """
-        if math.isclose(self.open_filled_amount, self.close_filled_amount):
+        if math.isclose(self.open_filled_amount, self.close_filled_amount) or (self.open_filled_amount > self.close_filled_amount):
```

Comment:
Hey @cardosofede, I found that this code might produce a bug where it can't close the position executor when the `open_filled_amount` is bigger than `close_filled_amount`, do you think `(self.open_filled_amount > self.close_filled_amount)` is good fix to make sure the position is closed properly in the case the fill amount is a bit bigger than expected?